src/client/Microsoft.Identity.Client/IAuthenticationScheme.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Identity.Client.Cache.Items;
+
+namespace Microsoft.Identity.Client
+{
+    /// <summary>
+    /// Used to modify the experience depending on the type of token asked. 
+    /// </summary>
+    internal interface IAuthenticationScheme
+    {
+        /// <summary>
+        /// Prefix for the HTTP header that has the token. E.g. "Bearer" or "POP"
+        /// </summary>
+        string AuthorizationHeaderPrefix { get; }
+
+        /// <summary>
+        /// Extra paramters that are added to the request to the /token endpoint
+        /// </summary>
+        /// <returns>Name and values of params</returns>
+        System.Collections.Generic.IDictionary<string, string> GetTokenRequestParams();
```

Comment:
add a using block to the top for System.Collections.Generic and just have IDictionary<string, string> here.